pkg/plugins/golang/v3/scaffolds/internal/templates/main.go

```diff
@@ -239,12 +239,13 @@ func main() {
 
 {{ if not .ComponentConfig }}
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
-		HealthProbeBindAddress: probeAddr,
-		LeaderElection:         enableLeaderElection,
-		LeaderElectionID:       "{{ hashFNV .Repo }}.{{ .Domain }}",
+		Scheme:                  scheme,
+		MetricsBindAddress:      metricsAddr,
+		Port:                    9443,
+		HealthProbeBindAddress:  probeAddr,
+		LeaderElection:          enableLeaderElection,
+		LeaderElectionNamespace: "{{ .ProjectName }}-system",
```

Comment:
We have more than one change/motivation here. 
Could you please create a PR for each case scenario? 
Also, can you please add a description of the issue faced to let us know why would you like to propose the change?

Note that the title of the pr is used to generate the changelogs so would be nice to have something that better clarifies the reason for the changes.

Reply:
Got it, raised a new PR: https://github.com/kubernetes-sigs/kubebuilder/pull/3021
PTAL thanks.